examples/step-57/step-57.cc

```diff
@@ -457,8 +457,8 @@ namespace Step57
                 local_rhs(i) +=
                   (-viscosity * scalar_product(grad_phi_u[i],
                                                present_velocity_gradients[q]) -
-                   phi_u[i] * (present_velocity_values[q] *
-                               present_velocity_gradients[q]) +
+                   phi_u[i] * (present_velocity_gradients[q] *
+                               present_velocity_values[q]) +
```

Comment:
Does this mean that we accidentally multiplied by (grad u)^T instead of (grad u)?